src/components/assessment/index.tsx

```diff
@@ -176,7 +176,12 @@ class Assessment extends React.Component<IAssessmentProps, State> {
         <p>
           You are about to finalise your submission for the{' '}
           {this.state.betchaAssessment.category.toLowerCase()}{' '}
-          <i>&quot;{this.state.betchaAssessment.title}&quot;</i>.
+          <i>
```

Comment:
Is this a formatter issue?

Reply:
Yup. I believe `yarn format` changed it as such